spring-boot/starter/src/main/java/org/togglz/spring/boot/actuate/TogglzEndpoint.java

```diff
@@ -53,4 +61,23 @@ public TogglzEndpoint(FeatureManager featureManager) {
         Collections.sort(features);
         return features;
     }
+
+    @WriteOperation
+    public TogglzFeature setFeatureState(@Selector String featureName, boolean enabled) {
+        final Feature feature = featureProvider.getFeatures().stream()
+                .filter(f -> f.name().equals(featureName))
+                .findFirst()
+                .orElseThrow(() -> new IllegalArgumentException("Could not find feature with name " + featureName));
+
+        FeatureState featureState = featureManager.getFeatureState(feature);
+
+        if (featureState.isEnabled() != enabled) {
+            featureState.setEnabled(enabled);
+            featureManager.setFeatureState(featureState);
+
+            return new TogglzFeature(feature, featureState);
+        }
+
+        return null;
```

Comment:
I would prefer if we still returned a `TogglzFeature` even if the state didn't change.

Reply:
I thought I'd be nice to return `null` here because in that case the actuator will respond with HTTP 204 instead of HTTP 200 which enables the client to deal with no actual changes (if someone would need that).

If you still prefer to return something else let's discuss that. In order to provide a more advanced HTTP response we need to add a `@EndpointWebExtension` I think.

Reply:
OK, now the endpoint always performs the set feature state operation and returns its result.